### PR TITLE
NF: Add dockerfile for singularity base

### PR DESCRIPTION
**Pull request by GuillaumeTh**

### CODE REVIEW

**Comment by jchoude** [comment]
Do you think you should add instructions on how to build this?

**Comment by GuillaumeTh** [comment]
Sure, I will add this. Do I add the process to push the docker on DockerHub or we keep this "private" ?

**Comment by jchoude** [comment]
I would keep that private

**Review by arnaudbore — APPROVED** [review]
Good :+1:

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
 # singularity-base-tractoflow
-Base container containing dependencies that do not frequently change for the Tractoflow singularity
+Base container containing dependencies that do not frequently change for the Tractoflow singularity.
+
+Build commands:
+
+Before build the Docker: `sudo docker image rm tractoflow:docker`
```
Comment (jchoude) [inline README.md:6]: `build` -> `building`

**Comment by jchoude** [comment]
I added just a comment, that can be fixed in another PR (since you told me you might need to create other PRs soon).

**Review by jchoude — APPROVED** [review]

**Pull request merged**